server/src/main/java/org/elasticsearch/common/logging/HeaderWarning.java

```diff
@@ -310,6 +310,14 @@ public static String getXOpaqueId() {
             .orElse("");
     }
 
+    public static String getTraceId() {
```

Comment:
Is this called anywhere? `TraceIdConverter` has its own implementation.

Reply:
Good find. I have removed that method.